recipes/mender-configure/debian-master/control

```diff
@@ -11,3 +11,17 @@ Depends: dbus, curl, mender-client (>=2.5.0)
 Description: Mender Configure
  Mender Configure is a Mender add-on which enhances the Mender Client providing an Update
  Module to configure the device
+
+Package: mender-configure-timezone
+Architecture: all
+Depends: mender-configure
+Description: Mender Configure Timezone
+ Mender Configure Timezone is an add-on to Mender Configure which enables configuration
+ of timezone on the device
+
+Package: mender-configure-demo
+Architecture: all
+Depends: mender-configure, mender-configure-timezone
```

Comment:
Does it really depend on `mender-configure-timezone`? I don't think so

Reply:
The UI will refer to broken functionality unless you install both. This was the thinking behind adding the dependency, to avoid user confusion. Do you think it's a good enough reason, or should we be more "technically correct" and remove it?

Reply:
Mmm, I think we should make it technically correct. Where you say "the UI" is really "the UI after onboarding", and in that case we have installed both via get.mender.io without this hacky dependency.

Reply:
I think it's not that hacky. The demo is the `mender-demo-raspberrypi-led` and the `timezone` scripts. The only reason that `timezone` is its own package is that it is useful outside the demo as well, otherwise it would have been inside the demo package. In that sense I guess the demo package could be considered more of a meta package, like `gnome`.

Reply:
okey. Makes sense